packages/rest/package.json

```diff
@@ -33,30 +33,30 @@
     "@aries-framework/node": "^0.1.0",
     "class-transformer": "0.5.1",
     "class-validator": "0.13.1",
-    "class-validator-jsonschema": "^3.1.0",
+    "class-validator-jsonschema": "^3.1.1",
     "cors": "^2.8.5",
-    "express": "^4.17.1",
-    "node-fetch": "^2.6.7",
+    "express": "^4.18.1",
+    "node-fetch": "2.6.7",
```

Comment:
Is it intentional to remove the `^`?

Reply:
node-fetch 3.0.0 has some issues, but just learned `^` doesn't update to major version so ill put it back :).